packaging/build_requirements.txt

```diff
@@ -0,0 +1,4 @@
+Jinja2>=2.9.3
+PyYAML>=3.13
+pytest
+sympy>=1.3,<1.6
```

Comment:
this is going to collide with my pr

Reply:
can you share your versions? then depending on who merges first will have to fix the file.

Reply:
I think Alessandro can rebase, shouldn't be big conflict